entries/s/store.steampowered.com.json

```diff
@@ -10,7 +10,7 @@
 		],
 		"documentation": "https://help.steampowered.com/en/faqs/view/06B0-26E6-2CF8-254C",
 		"recovery": "https://help.steampowered.com/en/wizard/HelpWithSteamIssue/?issueid=809",
-		"notes": "A phone number is required as a fallback for 2FA.",
+		"notes": "A phone number is required as a fallback for TOTP 2FA.",
```

Comment:
I think it doesn't make sense to refer to "TOTP 2FA" if the two options listed are `email` and `custom-software`.

Reply:
Updated wording, is that better?